gradle-consistent-versions-idea-plugin/src/test/resources/lexerTests/24bad4b06b90bba33694afd3e19b04a056cdbf8098e1eca917b90661a5999666

```diff
@@ -0,0 +1,18 @@
+VersionPropsTokenType.GROUP_PART has
```

Comment:
Can you put the test input at the top of the file? Otherwise it's hard to see what the input was.

Reply:
done :)